spec/unit/xlsx/xform/book/defined-name-xform.spec.js

```diff
@@ -8,17 +8,17 @@ var expectations = [
   {
     title: 'Defined Names',
     create: function() { return new DefinedNameXform(); },
-    preparedModel: {name: 'foo', ranges: ['bar!$A$1:$C$1']},
+    preparedModel: {name: 'foo', ranges: ['bar!$A$1:$C$1'], formula: ''},
```

Comment:
input data in tests should not be modified, it causes lost possibility to test backward compatibility